docker-pokedex

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env bash
-
-docker-compose run --rm pokedex "$@"
```

Comment:
It looks like maybe you moved this to `bin/docker-pokedex` but forgot to `git add` the new file.